lib/amazonka-core/src/Amazonka/Data/Path.hs

```diff
@@ -59,6 +61,9 @@ type RawPath = Path 'NoEncoding
 
 type EscapedPath = Path 'Percent
 
+newtype TwiceEscapedPath = TwiceEscapedPath (Path 'Percent)
```

Comment:
I think we want to add an additional `TwicePercent` constructor to `data Encoding`, so this can become a type alias for consistency.

Reply:
Wait, nevermind. That doesn't make sense.

Reply:
I went through the exact same sequence of thoughts, haha.